test/performance/search/index_construction_benchmark.cpp

```diff
@@ -0,0 +1,213 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/alphabet/aminoacid/aa27.hpp>
+#include <seqan3/search/fm_index/all.hpp>
+#include <seqan3/test/performance/sequence_generator.hpp>
+#include <seqan3/test/seqan2.hpp>
+#include <seqan3/range/views/persist.hpp>
+#include <seqan3/range/views/rank_to.hpp>
+#include <seqan3/range/views/to.hpp>
+#include <seqan3/range/views/to_rank.hpp>
+
+#if SEQAN3_HAS_SEQAN2
+#include <seqan/index.h>
+#endif
+
+static void arguments(benchmark::internal::Benchmark* b)
+{
+    for (int32_t length : {50, 5000, 50'000})
+    {
+        for (int32_t collection_size : {5})
+        {
+            b->Args({length, collection_size});
+        }
```

Comment:
Why do we need this loop? Are you planning to add more values later?

Reply:
Yes, I planned to; I think I might add a high number, like 100'000 to have some comparison for something-not-genomic.